### PR TITLE
doc: net: qemu: minor fixes (socat)

### PR DESCRIPTION
**Pull request by ghost**
Qemu Networking:
* The qemu networking docs did not properly document preliminaries.
* Persistent commands should be started in the background when started in a single terminal.

### CODE REVIEW

**Review by pdgendt — APPROVED** [review]

File: doc/connectivity/networking/qemu_setup.rst [hunk 1]
```diff
@@ -247,9 +248,8 @@ Terminal #1:
 
 .. code-block:: console
 
-   socat PTY,link=/tmp/slip.devMAIN UNIX-LISTEN:/tmp/slip.sockMAIN
-   $ZEPHYR_BASE/../net-tools/tunslip6 -t tapMAIN -T -s /tmp/slip.devMAIN \
-        2001:db8::1/64
+   socat PTY,link=/tmp/slip.devMAIN UNIX-LISTEN:/tmp/slip.sockMAIN &
+   sudo $ZEPHYR_BASE/../net-tools/tunslip6 -t tapMAIN -T -s /tmp/slip.devMAIN 2001:db8::1/64 &
```
Comment (jukkar) [inline doc/connectivity/networking/qemu_setup.rst:252]: By default the net-tools project is created under `tools/` directory so this should be changed to
```
sudo $ZEPHYR_BASE/../tools/net-tools/tunslip6 -t tapMAIN -T -s /tmp/slip.devMAIN 2001:db8::1/64 &
```

File: doc/connectivity/networking/qemu_setup.rst [hunk 2]
```diff
@@ -258,7 +258,6 @@ Terminal #2:
 
 .. code-block:: console
 
-   socat PTY,link=/tmp/slip.devOTHER UNIX-LISTEN:/tmp/slip.sockOTHER
-   $ZEPHYR_BASE/../net-tools/tunslip6 -t tapOTHER -T -s /tmp/slip.devOTHER \
-        2001:db8::1/64
+   socat PTY,link=/tmp/slip.devOTHER UNIX-LISTEN:/tmp/slip.sockOTHER &
+   sudo $ZEPHYR_BASE/../net-tools/tunslip6 -t tapOTHER -T -s /tmp/slip.devOTHER 2001:db8::1/64 &
```
Comment (jukkar) [inline doc/connectivity/networking/qemu_setup.rst:262]: ditto

**Review by jukkar — COMMENTED** [review]
Minor nit about the path

File: doc/connectivity/networking/qemu_setup.rst (hunk 1 above)
Comment (fgrandel) [reply doc/connectivity/networking/qemu_setup.rst:252]: Hrm, are you sure? This doesn't seem to fit the assumptions of the scripts in net_tools, see e.g. 

https://github.com/zephyrproject-rtos/net-tools/blob/7c7a856814d7f27509c8511fef14cec21f7d0c30/loop-slip-tap.sh#L22

```
for tunslip in ./tunslip6 "${0%/*}/tunslip6"  $ZEPHYR_BASE/../net-tools/tunslip6
```

Also compare https://docs.zephyrproject.org/latest/connectivity/networking/qemu_802154_setup.html
```
cd $ZEPHYR_BASE/../net-tools
```

Comment (jukkar) [reply doc/connectivity/networking/qemu_setup.rst:252]: `west.yml` has this
```
    - name: net-tools
      revision: 7c7a856814d7f27509c8511fef14cec21f7d0c30
      path: tools/net-tools
      groups:
        - tools
```
So it downloads net-tools into `tools/net-tools` directory. The directory was different when the documentation or tun script was created and should be updated.

Comment (fgrandel) [reply doc/connectivity/networking/qemu_setup.rst:252]: Is it ok, if we do this in a separate PR? Seems unrelated to this change and would probably something bigger. But I can go about it directly afterwards.

Comment (jukkar) [reply doc/connectivity/networking/qemu_setup.rst:252]: Of course, I wasn't planning to imply that the docs needs to fixed in this PR or even by you, just a note that we should fix the paths.

Comment (fgrandel) [reply doc/connectivity/networking/qemu_setup.rst:252]: See https://github.com/zephyrproject-rtos/net-tools/pull/75

Comment (fgrandel) [reply doc/connectivity/networking/qemu_setup.rst:252]: ...and https://github.com/zephyrproject-rtos/zephyr/pull/77878

**Review by rlubos — APPROVED** [review]

**Comment by fgrandel** [comment]
Assigned to @jukkar as the net-tools maintainer as these are actually changes referring to net-tools (nothing specifically to do with qemu).

**Review by jukkar — APPROVED** [review]

**Pull request merged**